install-dev/upgrade/php/create_multistore.php

```diff
@@ -44,7 +44,7 @@ function create_multistore()
     $all_themes_dir = _PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'themes';
     $themes = scandir($all_themes_dir);
     foreach ($themes as $theme) {
-        if (!is_file($all_themes_dir.DIRECTORY_SEPARATOR.$theme) && is_dir($all_themes_dir.DIRECTORY_SEPARATOR.$theme.DIRECTORY_SEPARATOR) && $theme[0] != '.' && $theme != 'prestashop'  && $theme != 'default-bootstrap') {
+        if (!is_file($all_themes_dir.DIRECTORY_SEPARATOR.$theme) && is_dir($all_themes_dir.DIRECTORY_SEPARATOR.$theme.DIRECTORY_SEPARATOR) && $theme[0] != '.' && $theme != 'prestashop'  && $theme != _PS_DEFAULT_THEME_NAME_) {
```

Comment:
I see that we kept comparison to 'prestashop' theme name. To be safe I would add a new case:
```
... && $theme != 'prestashop'  && $theme != 'default-bootstrap'  && $theme != 'classic'
```

It looks like we want to avoid default theme since 1.4 or something like that :confused: